src/PdfReader/index.tsx

```diff
@@ -20,8 +20,15 @@ type PdfState = ReaderState & {
   // if pageNumber is -1, we will navigate to the end of the
   // resource once it is parsed
   pageNumber: number;
+  scale: number;
+  pdfHeight: number;
+  pdfWidth: number;
+  pageHeight: number | undefined;
+  pageWidth: number | undefined;
 };
 
+pdfjs.GlobalWorkerOptions.workerSrc = `pdf.worker.min.js`;
```

Comment:
this is necessary to get the worker functional.  I'm not sure if there's a more elegant way to do this

Reply:
I'm hoping we don't have to actually copy the file from there to our repo... but we can do this for now